src/compiler/compile/css/Selector.ts

```diff
@@ -350,7 +350,7 @@ function attribute_matches(node: CssNode, name: string, expected_value: string,
 	const attr = node.attributes.find((attr: CssNode) => attr.name === name);
 	if (!attr) return false;
 	if (attr.is_true) return operator === null;
-	if (!expected_value) return true;
+	if (expected_value === null) return true;
```

Comment:
Do we need to handle undefined also?

```suggestion
	if (expected_value == null) return true;
```

Reply:
Sorry, totally missed your comment!

So, IDK, but when running the whole test suite, `expected_value` is never anything else than `null` | `string`.